BirthdayGreetingsTest/api.js

```diff
@@ -89,10 +84,10 @@ app.patch("/birthdays/update-friend/:email", (req, res) => {
   const searchEmail = req.params.email;
   updateBirthdayFriendInDB(lastName, firstName, dateOfBirth, email, searchEmail)
     .then((result) => {
-      res.status(200).send(result);
+      res.status(204).send(result);
```

Comment:
For an API a 200 is more appropriate here 👍

Reply:
I'll change it, but why is it when I look this up on google some results say 204 is correct? haha